packages/components/src/logos/index.tsx

```diff
@@ -23,27 +23,37 @@ export { ReactComponent as HiveCombinationMark } from './hive-combination-mark.s
 export { ReactComponent as GraphQLFoundationLogo } from './graphql-foundation.svg';
 
 export interface LettermarkLogoProps extends HTMLAttributes<HTMLElement> {}
-const createLettermarkLogo = (text: string) => {
+
+const HOCLettermarkLogo = (text: string) =>
   function LettermarkLogo(props: LettermarkLogoProps) {
     return (
       <span
         role="img"
         {...props}
         className={cn(
-          'inline-flex items-center justify-center text-sm font-medium uppercase leading-5',
+          'inline-flex items-center justify-center text-sm font-medium leading-5',
           props.className,
         )}
       >
         {text}
       </span>
     );
-  }
-  LettermarkLogo.displayName = 'LettermarkLogo';
-  return LettermarkLogo;
-};
+  };
 
-export const InspectorLettermark = createLettermarkLogo('I');
-export const SofaLettermark = createLettermarkLogo('So');
-export const GraphQLESlintLettermark = createLettermarkLogo('Esl');
-export const EnvelopLettermark = createLettermarkLogo('E');
-export const ScalarsLettermark = createLettermarkLogo('S');
+export const InspectorLettermark = HOCLettermarkLogo('INS');
+export const SofaLettermark = HOCLettermarkLogo('SOF');
+export const GraphQLESlintLettermark = HOCLettermarkLogo('ESL');
+export const EnvelopLettermark = HOCLettermarkLogo('ENV');
+export const ScalarsLettermark = HOCLettermarkLogo('SCL');
+export const ConductorLettermark = HOCLettermarkLogo('CON');
+export const StitchingLettermark = HOCLettermarkLogo('STI');
+export const ToolsLettermark = HOCLettermarkLogo('TLS');
+export const ModulesLettermark = HOCLettermarkLogo('MOD');
+export const ConfigLettermark = HOCLettermarkLogo('CFG');
+export const FetsLettermark = HOCLettermarkLogo('FTS');
+export const AngularLettermark = HOCLettermarkLogo('ANG');
+export const KitQLLettermark = HOCLettermarkLogo('KQL');
+export const WSLettermark = HOCLettermarkLogo('WS');
+export const SSELettermark = HOCLettermarkLogo('SSE');
+export const HeltinLettermark = HOCLettermarkLogo('HLT');
+export const WhatsAppLettermark = HOCLettermarkLogo('WAP');
```

Comment:
```suggestion
export const WhatsAppLettermark = HOCLettermarkLogo('WHA');
```

Okay, this is the most important comment I left in review since I had found an infinite money exploit in a billing system about a year ago.

We cannot ship capitalized "WAP" to prod. For the love of god, no.
It's very healthy you're not familiar with modern pop-culture to automatically read it as something else as "WhatsApp".

https://www.youtube.com/watch?v=hsm4poTWjMs

It's not a good song. Just trust me, accept the suggestion, and don't listen to it.

Reply:
Thank you! Really appreciated for this catch! 🤣 😍